tests/e2e/ctl_v2_test.go

```diff
@@ -30,6 +30,8 @@ func TestCtlV2SetClientTLS(t *testing.T) { testCtlV2Set(t, &configClientTLS, fal
 func TestCtlV2SetPeerTLS(t *testing.T)   { testCtlV2Set(t, &configPeerTLS, false) }
 func TestCtlV2SetTLS(t *testing.T)       { testCtlV2Set(t, &configTLS, false) }
 func testCtlV2Set(t *testing.T, cfg *etcdProcessClusterConfig, quorum bool) {
+	skipInShortMode(t)
```

Comment:
For the purpose of skipping e2e tests, is it sufficient to add `skipInShortMode(t)` to https://github.com/etcd-io/etcd/blob/01bd0d468d3eebd2825390fc7bcd17a5ec57d389/tests/e2e/cluster_test.go#L144:6? If so, the code is cleaner.

Reply:
This method does not have access to testing.T (nor testing.TB). 

If you prefer I can add the param (t) to the O(10) invocations of newEtcdProcessCluster(t, ...)

Reply:
Oh I see, so it is comparable lines of code changes. However I still prefer to pass testing.TB to newEtcdProcessCluster() due to better code readability and maintainability. If later some other developer contribute a new e2e test, it is guaranteed to be skipped in short mode. What do you think?

Reply:
SGTM. On it.